.github/workflows/issue-handling.yaml

```diff
@@ -28,7 +28,25 @@ jobs:
           project-url: https://github.com/orgs/timescale/projects/128
           github-token: ${{ secrets.ORG_AUTOMATION_TOKEN }}
           labeled: continuous_aggregate
-  
+
+  close-issue:
+    name: Issue is closed
+    runs-on: ubuntu-latest
+    if: github.event_name == 'issues' && github.event.action == 'closed'
+    steps:
+      - uses: leonsteinhaeuser/project-beta-automations@v2.0.0
+        with:
+          gh_token: ${{ secrets.ORG_AUTOMATION_TOKEN }}
+          organization: timescale
+          project_id: 55
+          resource_node_id: ${{ github.event.issue.node_id }}
+          status_value: 'Done'
```

Comment:
I am not sure how this action behaves if the issue is not already present on the board. I suggest executing this step only if the bug label is present to avoid adding all closed issues to the bug board or getting failures from the action.

Reply:
Done.